### PR TITLE
Fix menu mobile

### LINKED ISSUE #64: Bug - Open Menu In the Mobile Display
When we in mobile display of media screen less than 600px approximately and try to open the menu, it appears near the content screen and the sum of the menu width and content width is more than the actual screen so the screen slides. Better to make the menu appear above the content screen in those screen sizes.

### PR DESCRIPTION
**Pull request by giladfuchs**
Resolve #64 
Bug - Open Menu In the Mobile Display #64

# Description
small css edit

## screenshots
 
Screen Shot 2023-10-06 at 17 37 56

Screen Shot 2023-10-06 at 17 39 31

### CODE REVIEW

File: src/pages/components/header/menu/menu.scss [hunk 1]
```diff
@@ -57,8 +57,12 @@
     }
 
     .main .header.open{
+        display: block;
+        position: absolute;
+        z-index: 500;
         max-width: 100%;
         padding: 30px;
+        margin-right: .51rem;
```
Comment (NoamGaash) [inline src/pages/components/header/menu/menu.scss:65]: why this is the right number?

Comment (giladfuchs) [reply src/pages/components/header/menu/menu.scss:65]: it's can be change to 0.5 rem.

Comment (NoamGaash) [reply src/pages/components/header/menu/menu.scss:65]: rem is related to the default font size, and I'm curios fo understand why this specific size has been chosen

Comment (giladfuchs) [reply src/pages/components/header/menu/menu.scss]: I'm use to work with rem ..
any way I change it to  px

Comment (NoamGaash) [reply src/pages/components/header/menu/menu.scss]: rem is fine - I'm just confused by the very specific number

**Comment by NoamGaash** [comment]
@giladfuchs
please use the devtool to test various devices

image

**Comment by giladfuchs** [comment]
this pr it's not about the nav button section.

it's about the burger section.
when you open it on mobile it's go to the right side and you can't see it
Screen Shot 2023-10-10 at 13 22 18
 
 cause the screen get extend and it's not sticky.
 
 so you need to go to the right to see it

Screen Shot 2023-10-10 at 13 24 13
and after you try to close it this is what you get

Screen Shot 2023-10-10 at 13 26 19

this pr fix it to look like that.

Screen Shot 2023-10-10 at 13 26 50

and after you close it. it back to normal!

**Comment by NoamGaash** [comment]
allright, I see - thank you
however, let's open another issue to make this menu really mobile-friendly :)

**Review by NoamGaash — APPROVED** [review]

**Pull request merged**